src/main/java/Lovie.java

```diff
@@ -0,0 +1,103 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Lovie {
+    public static void main(String[] args) {
+        Scanner inputGetter = new Scanner(System.in);   
+        String input = new String(); 
+        ArrayList<Task> tasksList = new ArrayList<Task>();   
+
+        String logo = 
+        "       ██╗░░░░░░█████╗░██╗░░░██╗██╗███████╗ \n" +
+        "       ██║░░░░░██╔══██╗██║░░░██║██║██╔════╝ \n" +
+        "       ██║░░░░░██║░░██║╚██╗░██╔╝██║█████╗░░ \n" +
+        "       ██║░░░░░██║░░██║░╚████╔╝░██║██╔══╝░░ \n" +
+        "       ███████╗╚█████╔╝░░╚██╔╝░░██║███████╗ \n" +
+        "       ╚══════╝░╚════╝░░░░╚═╝░░░╚═╝╚══════╝ \n\n"; 
+        System.out.println("        Hey hey! My name is \n\n" + logo);
+        System.out.println("        How can I help you today?\n");
+        System.out.println("        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~" + 
+                "\n\n");
+
+        while (true) {
```

Comment:
try your best to avoid too many nested loops/ or if/else statements, because it might slow down your programme.